pyEpiabm/pyEpiabm/tests/test_sweep/test_spatial_sweep.py

```diff
@@ -64,6 +67,7 @@ def test__call__(self, mock_inf, mock_force, mock_poisson, mock_dist,
         self.microcell_susc.add_people(1)
```

Comment:
Adding the `microcell_susc` to `self.pop` could be done in `setUpClass` if you want to use it in multiple tests - best not to change anything in self in the tests as unittest doesn't guarantee it will run the tests in the order here! 

In general, probably best to avoid modifying anything in self inside a test - you can access attributes of self, but copy them if you want to modify them! Unit tests should run separately, and independent of each other (whereas the second test in this file requires the first to have modified self)

Reply:
But one of the tests needs to catch the case when the population only has one cell.